services/ansibler/server/loadbalancers.go

```diff
@@ -156,7 +156,7 @@ func teardownLoadBalancers(clusterName string, info *LBInfo, attached bool) (str
 	k8sDirectory := filepath.Join(baseDirectory, outputDirectory, fmt.Sprintf("%s-%s-lbs", clusterName, utils.CreateHash(utils.HashLength)))
 
 	if err := generateK8sBaseFiles(k8sDirectory, info); err != nil {
-		log.Err(err)
+		log.Error().Err(err)
```

Comment:
I am not sure this would send the message. I would expect some `Msg()` call at the end, i.e. like you did it here
```go
logger.Err(err).Msgf("Error encountered while installing node requirements")
```